src/WebJobs.Script/ExtensionBundle/ExtensionBundleManager.cs

```diff
@@ -246,8 +246,13 @@ private async Task<string> GetLatestMatchingBundleVersionAsync(HttpClient httpCl
             return matchingBundleVersion;
         }
 
-        private static string FindBestVersionMatch(VersionRange versionRange, IEnumerable<string> versions)
+        internal static string FindBestVersionMatch(VersionRange versionRange, IEnumerable<string> versions)
         {
+            versionRange = new VersionRange(versionRange.MinVersion,
```

Comment:
Where is the versionRange parameter coming in from (caller of this method)? Would it make more sense to do this sooner?

Reply:
This method is called by two flows, locating extension bundle on local machine and location extension bundle on cdn for download. Doing it sooner would mean we would have to make a change in 2 places.

Reply:
thanks for the info!